addons/account_edi/tests/common.py

```diff
@@ -34,6 +34,12 @@ def setUpClass(cls, chart_template_ref=None, edi_format_ref=None):
     def edi_cron(self):
         self.env['account.edi.document'].sudo().with_context(edi_test_mode=True).search([('state', 'in', ('to_send', 'to_cancel'))])._process_documents_web_services()
 
+    def create_invoice(self):
+        invoice = self.env['account.move'].with_context(default_move_type='in_invoice').create({})
```

Comment:
`self.env['account.move'].create({'move_type': 'in_invoice'})` ?
Also, I would like to rename the method: `_create_empty_vendor_bill`